### PR TITLE
Adding Test and Modifying Models  for Tests

### PR DESCRIPTION
**Pull request by TurtleSquad007**

### CODE REVIEW

**Review by giancarlopernudisegura — COMMENTED** [review]
What is this empty class?

**Comment by TurtleSquad007** [comment]
It's what will eventually becomes the user tests. I stopped working on it when class started so it's empty.

File: code/app/src/main/java/com/example/experiment_automata/backend/users/User.java [hunk 1]
```diff
@@ -121,46 +148,13 @@ public void onFailure(@NonNull Exception e) {
                 });
     }
 
-    /**
-     * Update the user information from the Firestore.
-     */
-    protected void updateFromFirestore() {
```
Comment (Pegmode) [inline code/app/src/main/java/com/example/experiment_automata/backend/users/User.java:127]: This is a major change. How did you end up handling this?

Comment (TurtleSquad007) [reply code/app/src/main/java/com/example/experiment_automata/backend/users/User.java:127]: Not sure what you mean. Like how did I remove the user from talking to firebase

File: code/app/src/main/java/com/example/experiment_automata/backend/users/UserManager.java [hunk 2]
```diff
@@ -27,9 +27,25 @@ private UserManager()
...
+            userManager = new UserManager();
+            userManager.getAllUsersFromFireStore();
+        }
+        else if (userManager == null && testMode)
+        {
+            userManager = new UserManager();
+        }
+
+
     public static UserManager getInstance() {
```
Comment (Pegmode) [inline code/app/src/main/java/com/example/experiment_automata/backend/users/UserManager.java:49]: This line is causing errors.

`Experiment-Automata\code\app\src\main\java\com\example\experiment_automata\backend\users\UserManager.java:49: error: illegal start of expression
    public static UserManager getInstance() {
    ^`

**Review by Pegmode — CHANGES_REQUESTED** [review]

**Review by Pegmode — APPROVED** [review]
Issues have been fixed

**Pull request merged**